cleanWeather-vue/1.md

```diff
@@ -60,6 +60,6 @@ One of the biggest benefits of using NativeScript is the fast development workfl
 
 > **TIP**:
 > * You can save your current Playground work by clicking the **Save** button at the top. This will generate a URL that you can bookmark for future use.
-> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app you can continue developing using [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) or the [NativeScript command-line interface](https://docs.nativescript.org/start/quick-setup).
+> * If you want to continue developing this app locally, you can use the **Download** button at the top of the screen. After you download the app you can continue by installing NativeScript command-line interface with `npm i nativescript -g` and running `tns preview` command to preview your work the same way you do it in Playground.
```

Comment:
[Suggestion] You can consider rephrasing this to:
After you download the app, you should install the NativeScript CLI by running the `npm i nativescript -g` command and then execute the `tns preview` command to preview your work, just as you do in the Playground.